paper.tex

```diff
@@ -169,6 +235,36 @@ \subsection{Approximation of the subdivision density integral}
     \label{fig:espc}
 \end{figure}
 
+\section{Error metrics for approximation by arcs}
+
+The problem of approximating a curve by a sequence of arc segments has extensive literature, but none of the published solutions are quite suitable for our application. The specific problem of approximating an Euler spiral by arcs is considered in \cite{Meek2004} using a ``cut then measure'' adaptive subdivision scheme, but their solution has poor quality; it scales as $O(1/n^2)$, while $O(1/n^3)$ is attainable. The result was improved ``slightly'' by \cite{Narayan2014}. The literature also contains optimal results, namely \cite{Maier2014} and \cite{Nuntawisuttiwong2021}, but at considerable cost; both approaches claim $O(n^2)$ time complexity. The through-line for all these results is that they are solving a harder problem, adopting the constraint that the generated arc sequence is $G^1$ continuous. While desirable for many applications, for simply rendering a stroked outline, it is not needed. Even with this constraint relaxed, the angle discontinuities of an arc approximation are tiny compared to the flattening to lines.
```

Comment:
> they are solving a harder problem, adopting the constraint that the generated arc sequence...

nit: Use `:` instead of comma after "they are solving a harder problem"?